core/tests/webdriverio_desktop/embedding.js

```diff
@@ -120,6 +116,12 @@ describe('Embedding', function() {
     '\'X-Frame-Options\' to \'deny\'.',
   ];
 
+  beforeAll(async() => {
+    explorationEditorPage = new ExplorationEditorPage.ExplorationEditorPage();
+    explorationEditorMainTab = explorationEditorPage.getMainTab();
+    explorationEditorSettingsTab = explorationEditorPage.getSettingsTab();
+  });
```

Comment:
Why it was shifted here?

Reply:
We cannot use a constructor outside beforeAll, beforeEach, it blockin webdriverio (We discussed this in starting when declaring AdminPage outside 'it' block was throwing error.